### PR TITLE
Add a print_bad alias for print_error

### PR DESCRIPTION
**Pull request by todb-r7**
Came up on Twitter, where Justin may have been trolling a little:

https://twitter.com/jstnkndy/status/798671298302017536

We have a `print_good` method, but not a `print_bad`, which seems a
little weird for Ruby -- opposite methods should be intuitive as Justin
is implying.

Anyway, I went with alias_method, thanks to the compelling argument at

https://github.com/bbatsov/ruby-style-guide#alias-method

...since Metasploit is all about the singleton, and didn't want to risk
some unexpected scoping thing.

So, TL;dr: This adds `print_bad` as an alias for `print_error` everywhere that `print_error` is defined. It also adds `vprint_bad` as an alias for `vprint_error`.

## Verification

- [ ] `echo "<ruby>" > /tmp/print.rc`
- [ ] `echo "print_good 'This is good';print_bad 'This is bad'" >> /tmp/print.rc`
- [ ] `echo "</ruby>" >> /tmp/print.rc`
- [ ] `./msfconsole -Lqr /tmp/print.rc -x exit`

- [ ] **Verify** that you see the lines printed with correct console coloring.

### CODE REVIEW

**Comment by h00die** [comment]
I can't +1 this enough. EVERY module I've ever written I've done print_bad and had to fix it.

**Comment by todb-r7** [comment]
There's a module that tries to exercise `print_bad` already:

```
(upstream-master) todb@mazikeen:~/git/metasploit-framework$ grep -r print_bad modules
modules/post/windows/gather/bitlocker_fvek.rb:        print_bad('Failed to generate FVEK, wrong recovery key?')
```

So, this PR will fix it. Adding the bug label.

**Comment by bcook-r7** [comment]
what a fortunate coincidence!

**Comment by jstnkndy** [comment]
And to think I was just poking fun, but this is logical and is a great addition :+1:

**Comment by egypt** [comment]
This would be a good candidate for a new committer to land. *cough* @h00die *cough*

**Comment by h00die** [comment]
i'll get to this today

**Comment by h00die** [comment]
```
msf > irb
[*] Starting IRB shell...

>> print_bad('test')
[-] test
=> nil
>> vprint_bad('verbose test')
NoMethodError: undefined method `vprint_bad' for #<Msf::Ui::Console::CommandDispatcher::Core:0x0055d0d21a6758>
Did you mean?  print_bad
	from (irb):2:in `cmd_irb'
	from /git/metasploit-framework/lib/rex/ui/text/irb_shell.rb:53:in `block in run'
	from /git/metasploit-framework/lib/rex/ui/text/irb_shell.rb:52:in `catch'
	from /git/metasploit-framework/lib/rex/ui/text/irb_shell.rb:52:in `run'
	from /git/metasploit-framework/lib/msf/ui/console/command_dispatcher/core.rb:911:in `cmd_irb'
	from /git/metasploit-framework/lib/rex/ui/text/dispatcher_shell.rb:430:in `run_command'
	from /git/metasploit-framework/lib/rex/ui/text/dispatcher_shell.rb:392:in `block in run_single'
	from /git/metasploit-framework/lib/rex/ui/text/dispatcher_shell.rb:386:in `each'
	from /git/metasploit-framework/lib/rex/ui/text/dispatcher_shell.rb:386:in `run_single'
	from /git/metasploit-framework/lib/rex/ui/text/shell.rb:203:in `run'
	from /git/metasploit-framework/lib/metasploit/framework/command/console.rb:48:in `start'
	from /git/metasploit-framework/lib/metasploit/framework/command/base.rb:82:in `start'
	from ./msfconsole:48:in `<main>'
```

was `vprint_bad` forgotten in irb?

**Comment by h00die** [comment]
# Release Notes

Adds aliases to `print_error` and `vprint_error` to `print_bad` and `vprint_bad`.  This makes the opposite of `print_good` more intuitive.

**Comment by todb-r7** [comment]
Thanks @h00die !
